### ISSUE TITLE
The removed permissions table is not correctly populated for supported and finalised/executed motions

### ISSUE THREAD

**Issue by rumzledz** [JoinColony/colonyCDapp#3904]
## Steps to reproduce

1. Give Fry admin permissions in General
2. Install the Reputation extension
3. Remove Fry's permissions via the Reputation decision method
4. Support and finalise the motion

## Expected behaviour

The Completed Action form correctly shows the table data for Fry's removed permissions

## Actual behaviour

The Completed Action form shows incorrect table data for Fry's removed permissions

## The Problem

I might be wrong but unlike doing this via the Permissions decision method, there is currently no accurate way to know what permissions were removed (turned to `false`) when a motion is finalised or executed.

| via Permissions | via Reputation |
| ---- | ---- |
| <img width="264" alt="Screenshot 2024-12-11 at 14 51 32" src="https://github.com/user-attachments/assets/a79eb452-b0bc-47e2-94b5-980fd6676eee"> | <img width="194" alt="Screenshot 2024-12-11 at 14 53 36" src="https://github.com/user-attachments/assets/c59fed19-38de-4153-8ffd-d39f3b44bd37"> |

Basically with Permissions, we can at least identify that "Oh, role 6 is now false so we know this role was the one that got removed!"

But with motions, everything is false 🫠 

## Potential Solutions

Might need to do something on the block-ingestor side. Like perhaps replicating the kind of boolean flags we set when this is done via Reputation (?)
